src/pages/Account/Venue/TemplateForm.tsx

```diff
@@ -73,6 +73,7 @@ const TemplateFormLeft: React.FC<TemplateFormLeftProps> = (props) => {
         </div>
       </div>
       <div className="page-container-left-bottombar">
+        <div />
```

Comment:
What does this do?

Reply:
'justify-content: space-between' to ensure the button is to the right

Reply:
Can't we have `justify-content: end`?

Reply:
the next screen has a 'go back' button on the left hand side